### ISSUE TITLE
AnyDesk starts but exits soon after (flatpak and RPM)

### ISSUE THREAD

**Issue by andreaippo** [flathub/com.anydesk.Anydesk#150]
Hi there, I love this app but today I had to give support after a long time without any requests, and when I tried to launch it I noticed it no longer works.

I prefer flatpaks whenever possible, but I also tried the RPM from the opensuse repo and I get the same behavior.

App is briefly listed in my System Monitor with 15-16MB of RAM used, then closes.

No UI/tray icon ever appears during this time.

Launching it from the CLI also doesn't say much, no errors are reported.

What can I do?

Thanks

Operating System: openSUSE Tumbleweed 20241112
KDE Plasma Version: 6.2.3
KDE Frameworks Version: 6.8.0
Qt Version: 6.8.0
Kernel Version: 6.11.7-1-default (64-bit)
Graphics Platform: Wayland
Processors: 16 × AMD Ryzen 7 7840U w/ Radeon 780M Graphics
Memory: 30.7 GiB of RAM
Graphics Processor: AMD Radeon 780M
Manufacturer: Framework
Product Name: Laptop 13 (AMD Ryzen 7040Series)
System Version: A7

```
❯ flatpak list | grep anydesk
AnyDesk com.anydesk.Anydesk     6.3.3   stable  system
```

**Comment by der-eismann** [comment]
Hey Andrea, if it happens with the RPM as well I'm afraid it's an issue with AnyDesk itself on your system and not the Flatpak specifically, so unfortunately I can't help you with that. You could try contacting AnyDesk support or write on Reddit.
You could try the newly released 6.4.0 RPM, I'm currently updating the Flatpak as well, maybe that will help.

**Comment by andreaippo** [comment]
Hi, thanks. It's indeed rather an issue with AnyDesk.

```
   info 2024-11-15 21:34:30.521         ad   main  21793  21793                         impl_selector - using avx2
 * * * * * * * * * * * * * * * * * *
   info 2024-11-15 21:34:30.521         ad   main  21793  21793                                  main - * AnyDesk Linux Startup *
   info 2024-11-15 21:34:30.521         ad   main  21793  21793                                  main - * Version 6.4.0 ((unknown) 1d6fef66806e5d14126bc45c1888351868d0eed5)
   info 2024-11-15 21:34:30.521         ad   main  21793  21793                                  main - * Build 20241105153419
   info 2024-11-15 21:34:30.521         ad   main  21793  21793                                  main - * Prefix ad
   info 2024-11-15 21:34:30.521         ad   main  21793  21793                                  main - * Copyright (C) 2024 AnyDesk Software GmbH *
   info 2024-11-15 21:34:30.521         ad   main  21793  21793                                  main - 
   info 2024-11-15 21:34:30.521         ad   main  21793  21793                                  main - Distribution Info: openSUSE Tumbleweed
   info 2024-11-15 21:34:30.521         ad   main  21793  21793                                  main - Found GLib Version: 2.82.
   info 2024-11-15 21:34:30.549         ad   main  21793  21793                       app.common_path - user_data_dir = /home/andrea/.anydesk, logged_in_user_data_dir = /home/andrea/.anydesk, current_home = /home/andrea

   info 2024-11-15 21:34:30.549         ad   main  21793  21793                       app.common_path - initialized user = /home/andrea/.anydesk, system = /etc/anydesk
   info 2024-11-15 21:34:30.550         ad   main  21793  21793             app.named_process_starter - Spawning new process with: UID: 1000, DPY: :0, ID: 3, SEAT: 930990432
   info 2024-11-15 21:34:30.551         ad   main  21793  21793                       app.common_path - user_data_dir = /home/andrea/.anydesk, logged_in_user_data_dir = /home/andrea/.anydesk, current_home = /home/andrea

   info 2024-11-15 21:34:30.551         ad   main  21793  21793                       app.common_path - initialized user = /home/andrea/.anydesk, system = /etc/anydesk
   info 2024-11-15 21:34:30.551         ad   main  21793  21793                                  main - Starting frontend process (21793).
explode 2024-11-15 21:34:30.551   frontend   main  21793  21793                   unix_base.x11_tools - Cannot open display
warning 2024-11-15 21:34:30.554         ad   main  21798  21798        base.signal.global.shared_file - open (/proc/1805/environ) failed (13): Permission denied 
   info 2024-11-15 21:34:30.577         ad   main  21798  21798                         impl_selector - using avx2
 * * * * * * * * * * * * * * * * * *
   info 2024-11-15 21:34:30.577         ad   main  21798  21798                                  main - * AnyDesk Linux Startup *
   info 2024-11-15 21:34:30.577         ad   main  21798  21798                                  main - * Version 6.4.0 ((unknown) 1d6fef66806e5d14126bc45c1888351868d0eed5)
   info 2024-11-15 21:34:30.577         ad   main  21798  21798                                  main - * Build 20241105153419
   info 2024-11-15 21:34:30.577         ad   main  21798  21798                                  main - * Prefix ad
   info 2024-11-15 21:34:30.577         ad   main  21798  21798                                  main - * Copyright (C) 2024 AnyDesk Software GmbH *
   info 2024-11-15 21:34:30.577         ad   main  21798  21798                                  main - 
   info 2024-11-15 21:34:30.577         ad   main  21798  21798                                  main - Distribution Info: openSUSE Tumbleweed
   info 2024-11-15 21:34:30.577         ad   main  21798  21798                                  main - Found GLib Version: 2.82.
   info 2024-11-15 21:34:30.606         ad   main  21798  21798                       app.common_path - user_data_dir = /home/andrea/.anydesk, logged_in_user_data_dir = /home/andrea/.anydesk, current_home = /home/andrea

   info 2024-11-15 21:34:30.606         ad   main  21798  21798                       app.common_path - initialized user = /home/andrea/.anydesk, system = /etc/anydesk
explode 2024-11-15 21:34:30.606       tray   main  21798  21798                   unix_base.x11_tools - Cannot open display
```

I thought Wayland was supported though?! Did they drop support for it? Do you use it under Wayland? I seem to recall I used to...

Thanks
